src/components/sound-card/sound-card.jsx

```diff
@@ -0,0 +1,32 @@
+import { useSounds } from "../../hooks/use-sounds";
+import purpleGoblin from "../../images/purple-goblin.png";
+
+export function SoundCard({ soundInfo }) {
+  const card = {
+    image: soundInfo?.photo,
+    title: soundInfo?.title,
+    audio: soundInfo?.audio,
+  };
+
+  const { playSound } = useSounds();
+
+  function handleSoundClick() {
+    playSound(
+      `${card.audio.startsWith("http") ? card.audio : "https://" + card.audio}`
```

Comment:
super nit: The template literal can be used just for when we need to create a string, ie: 
```
card.audio.startsWith("http") ? card.audio : `https://${card.audio}`
```